pkg/storage/mongo/getter.go

```diff
@@ -24,9 +24,17 @@ func (s *ModuleStore) Get(ctx context.Context, module, vsn string) (*storage.Ver
 		}
 		return nil, err
 	}
+
+	zipName := s.gridFileName(module, vsn)
+	fs := s.s.DB(s.d).GridFS("fs")
+	f, err := fs.Open(zipName)
```

Comment:
dont you need to close the file? i see gridfs has exported method for that

Reply:
thanks @marwan-at-work missed that.

Reply:
i'm thinking whether or not to rename ZIp -> ZipReadCloser 
so it's obvious you have to close it

Reply:
Zip *is* a ReadCloser. As long as your code editor shows you the type, then it should be clear. But if you think being more verbose is better, happy to change it :)

Reply:
no i guess there's no need. it's not a common practice so lets avoid it.

Reply:
@michalpristas sounds good. Let me know if this PR is good on your side : )